### ISSUE TITLE
Option to remove grepWin context menu integration

### ISSUE THREAD

**Issue by Richard-L** [stefankueng/grepWin#442]
First off want to say that grepWin is absolutely wonderful, and has enabled a very niche requirement I've had in my workflow that I've searched high and low for across different apps, powershell scripts, etc. So thank you for making grepWin 🙏 

I have one issue in that I cannot disable the context menu integration in these two cases on Win11:
![image](https://github.com/stefankueng/grepWin/assets/4934209/7eea7c18-b517-4a96-8479-99387c47eb8b)
![image](https://github.com/stefankueng/grepWin/assets/4934209/cf2bb546-cd60-4d89-9cde-a1b3aa06f4cf)

I will try to hack these away with third party context menu editing tools, but truthfully it's necessary that grepWin allows user control over this. At least in the installer, but better yet inside settings.

Thanks for considering.

**Comment by stefankueng** [comment]
if you don't want/need the context menu, just use the portable version instead of the msi installer.

**Comment by Richard-L** [comment]
Thanks for responding.

I'm quite happy to have the tool installed properly. It helps to make Windows find it as a recognised app in things like quick search from the task bar, etc.

**Comment by outlawzgosu** [comment]
Can this be added to some kind of to-do list? There's no autoupdate on the portable version.

**Comment by dlundell-cissi** [comment]
For the modern context menu

Windows Registry Editor Version 5.00

[-HKEY_CLASSES_ROOT\PackagedCom\Package\7BE7C3C4-6740-4AB4-9C5B-DD64067515BF_2.1.9.1420_x64__rn3ewnym04k8y\Class]
[-HKEY_CLASSES_ROOT\PackagedCom\Package\7BE7C3C4-6740-4AB4-9C5B-DD64067515BF_2.1.9.1420_x64__rn3ewnym04k8y\Server]
[-HKEY_CLASSES_ROOT\PackagedCom\ClassIndex\{3C557AFF-6181-4BBC-937D-E2FE8844DD49}]

But it shouldn't be there in the first place without asking.

**Comment by outlawzgosu** [comment]
> For the modern context menu
> 
> Windows Registry Editor Version 5.00
> 
> [-HKEY_CLASSES_ROOT\PackagedCom\Package\7BE7C3C4-6740-4AB4-9C5B-DD64067515BF_2.1.9.1420_x64__rn3ewnym04k8y\Class] [-HKEY_CLASSES_ROOT\PackagedCom\Package\7BE7C3C4-6740-4AB4-9C5B-DD64067515BF_2.1.9.1420_x64__rn3ewnym04k8y\Server] [-HKEY_CLASSES_ROOT\PackagedCom\ClassIndex{3C557AFF-6181-4BBC-937D-E2FE8844DD49}]
> 
> But it shouldn't be there in the first place without asking.

It's not that it's there without asking, it's that we can't remove it if we no longer want it there.

**Comment by Manhunter07** [comment]
+1 it bloats the context menu and serves no purpose in most cases. If I want to use it, i run it myself.

**Comment by Ali229** [comment]
+1 please remove this or at least ask the user before adding it.

**Comment by lefth** [comment]
Agreed. It's kind of scary to need to edit the registry to remove the context menu shell extension. It would be much better in the settings or at least as an option in the installer.

Edit: it didn't work anyway. I'm uninstalling and will install the portable version.

**Comment by mrx23dot** [comment]
I have the other issue, an extra batch script or menu option to register portable install for context menu would be great.
